css/src/atomics/typography.scss

```diff
@@ -82,6 +82,10 @@
 	text-decoration: underline;
 }
 
+.text-decoration-none {
+	text-decoration: none;
```

Comment:
Does the `!important` need to be added here?

Reply:
Yes! In fact, it seems like we forgot to add !important to the two above.

Important is used with atomic classes to ensure that single class selectors will be effective even against more specific component classes.

Reply:
(I've added it to the other here)